python/mlx/nn/layers/transformer.py

```diff
@@ -157,7 +157,7 @@ def __call__(self, x, memory, x_mask, memory_mask):
         x = x + y
 
         y = self.ln2(x)
-        y = self.cross_attention(x, memory, memory, memory_mask)
```

Comment:
Great catch! Thank you! This little typos are often the hardest to find :).

Reply:
💯